internal/contour/handler.go

```diff
@@ -65,6 +65,9 @@ type EventHandler struct {
 	// seq is the sequence counter of the number of times
 	// an event has been received.
 	seq int
+
+	// Converter handles conversion from unstructured to a type
+	converter k8s.Converter
```

Comment:
I kinda feel that the field and the type both have missed the opportunity to obviate the comment. If k8s.Converter was called k8s.UnstructuredConverter then maybe the commend wouldn't be necessary.